test/unit/org/apache/cassandra/db/compaction/LeveledCompactionStrategyTest.java

```diff
@@ -695,10 +695,10 @@ public void singleTokenSSTableTest()
     }
 
     @Test
-    public void randomMultiLevelAddTest()
+    public void testRandomMultiLevelAdd()
     {
         int iterations = 100;
-        int levelCount = 8;
+        int levelCount = 9;
```

Comment:
In fact there are 9 levels, and it is better to test all levels (including the highest one).